docsrc/imap/reference/manpages/systemcommands/squatter.rst

```diff
@@ -237,6 +237,18 @@ Options
     Recursively create indexes for all sub-mailboxes of the user,
     mailboxes or mailbox prefixes given as arguments.
 
+.. option:: -s, --squat-skip[=delta]
+
+    Skip mailboxes that have not been modified since last index. This is
+    achieved by comparing the last modification time of a mailbox to
+    the last time the squat index of this mailbox got updated. If the
+    mailbox modification time (plus delta) is less than the squat
+    index modification time, then the mailbox is skipped. The optional
+    argument value delta is defined in seconds. It must be non-zero,
```

Comment:
From the `case 's':` switch block below, it looks like it also has to be positive?  I'm not sure if a negative delta would ever be useful

Reply:
No, a negative delta is never useful.